site/zenodo_rdm/legacy/requests/utils.py

```diff
@@ -82,3 +83,26 @@ def submit_record_upgrade_request(record, uow, comment=None):
     return current_requests_service.execute_action(
         system_identity, request_item._request.id, "submit", data=comment, uow=uow
     )
+
+
+@unit_of_work()
+def submit_community_manage_record_request(user_id, uow, comment=None):
+    """Create and submit a CommunityManageRecord request."""
+    type_ = current_request_type_registry.lookup(CommunityManageRecord.type_id)
+    receiver = ResolverRegistry.resolve_entity_proxy({"user": user_id}).resolve()
+    expires_at = datetime.utcnow() + timedelta(weeks=20)
```

Comment:
Lars' comment about the expiration date not being during Christmas holidays: just added 5 months, so that it will just be the end of January